src/libs/actions/IOU.js

```diff
@@ -152,6 +152,19 @@ function rejectTransaction({
         });
 }
 
+/**
+ * Sets IOU'S selected currency
```

Comment:
Is this only used for IOU? If so, then the name should reflect that and I don't think `myPersonalDetails` is the correct place for this. If not, then we should not mention an IOU in this comment.

Reply:
Done!